README.md

```diff
@@ -185,7 +185,7 @@ When setting 'useFmtOnly' property to 'true' for establishing a connection or cr
 <dependency>
 	<groupId>com.microsoft.sqlserver</groupId>
 	<artifactId>mssql-jdbc</artifactId>
-	<version>8.2.2.jre13</version>
+	<version>8.3.0.jre14-preview</version>
 </dependency>
 
 <dependency>
```

Comment:
Add 8.3 to the end of line 228 (should we even continue adding new ones at the end of that sentence tho?)

Reply:
I think we can stop adding new versions.